triplea_maps.yaml

```diff
@@ -88,10 +88,10 @@
   version: 1
   img: http://tripleamaps.sourceforge.net/images/TripleA_total_world_war_mini.png
   description: |
-    <br><em><b><font size="6">Created by Rolf Larsson and Hepster</font></b></em>
+    <br><em><b style="font-size: 6px;">Created by Rolf Larsson and Hepster</b></em>
```

Comment:
I should have clarified in my previous comment that the equivalence between `xx-large` and `6` was on Firefox.  As this is a browser-specific mapping, the Swing renderer can use a different mapping, and it apparently does by mapping `6` to `x-large` as shown below:

![font-size](https://cloud.githubusercontent.com/assets/4826349/26757335/7ec20084-4886-11e7-950d-f960805599bc.png)